src/runtime/micro/host_driven/utvm_device_dylib_redirect.c

```diff
@@ -32,10 +32,10 @@ extern "C" {
 #include <stdint.h>
 #include <stddef.h>
 
-void *(*TVMBackendAllocWorkspace_)(int, int, uint64_t, int, int) =
-    (void *(*)(int, int, uint64_t, int, int)) NULL;
-int (*TVMBackendFreeWorkspace_)(int, int, void*) = (int (*)(int, int, void*)) NULL;
-void (*TVMAPISetLastError_)(const char*) = (void (*)(const char*)) NULL;
+// TODO(areusch): compiler errors say volatile qualifier is discarded. should we just get rid of em?
```

Comment:
add back `weberlo` to the TODOs you replaced with `areusch` (e.g., `TODO (weberlo, areusch)`), so I can still grep for them

Reply:
done